### ISSUE TITLE
Export doesn't include parenthesis {} in latex commands

### ISSUE THREAD

**Issue by Flo472** [retorquere/zotero-better-bibtex#1529]
**Report ID:**
7RYCY59R-euc
**Exporter used:**
5.2.24
**Expected behavior:**
in previous versions it exported {} parenthesis in latex commands.
**Actual behavior:**
my .bib file doesn't compile anymore, but older versions do. The obvious difference between older exported and new exported .bib files is that BBT doesn't export latex commands with {} parenthesis anymore

**Comment by retorquere** [comment]
> 7. you have included an error-report ID here generated by reproducing the problem, selecting the problematic reference(s), right-clicking, and submitting an BBT error report from that popup menu
> 8. you have included the actual output you got from exporting the items you sent in the debug log, not simplified fictional output, and a sample of you you want it to look, again based on the actual items under consideration

7RYCY59R-euc was sent from the Help menu, not as described by point 7, and I need a sample of the output of the references you selected to generate the debug log, and a sample, not a description, of what you expect them to look like.

**Comment by retorquere** [comment]
Also I need to know which exporter you used (better bibtex or better biblatex).

**Comment by retorquere** [comment]
I really need more information.

**Comment by retorquere** [comment]
It may be that you're referring to `{}` at the end of latex commands without parameters, which is indeed new in 5.2.22, but without an actual problematic item I have no idea where the problem occurs. If there's a bug in this then it's a critical one, so I'd prefer to fix this before it bites a lot of people, but without the information mentioned in points 7 and 8, and information on whether this is a bibtex or biblatex export (although I should be able to tell from the output of point 8), there's no way for me to even start on a diagnosis.

**Comment by Flo472** [comment]
I tried hard to reproduce the problem in a minimum example and/or trace it to a specific reference that I use, but I couldn't reproduce the problem. I tried different things with my Latex file, and it seems as if it could have been also an encoding issue. Now it works though and I have no clue why. Also if I re-export from Zotero it works without problem now.

Sorry for not following the points 7&8, I tried afterwards, but as said, couldn't reproduce the problem or trace it to a specfic reference. I export to Biblatex. But I did mean the {} at the end of some latex commands which are not there anymore. That was only an obvious difference, but now it works also with version 5.2.24 for me.

Thanks for your quick reply! I really enjoy working with BetterBibTex and Zotero, and it was the first issue I had, ever, great work! (I used Mendeley before, and it's beyond comparison)
